README.md

```diff
@@ -290,16 +290,13 @@ cp ~/.ani-cli/ani-cli ~/.local/bin/
 ```
 echo '[Desktop Entry]
 Encoding=UTF-8
-Version=4.0
 Type=Application
-Exec=konsole -e ani-cli
-Name=ani-cli' > ~/.local/share/applications/ani-cli.desktop
+Exec=bash -c "source $HOME/.'$(echo $SHELL | sed -nE "s|.*/(.*)\$|\1|p")'rc && konsole --fullscreen -e ani-cli"
```

Comment:
any chance of cleaning this up further?

Reply:
Quite unlikely, but if i find a better solution, ill implement it.